### DISCUSSION TITLE
[VOTE] Release Apollo version 1.8.2

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#3739] [General]
Hi All,

This is a call for vote to release Apollo version 1.8.2

Release notes:

 * https://github.com/ctripcorp/apollo/releases/tag/v1.8.2

Release Candidate:

 * https://github.com/ctripcorp/apollo/releases/download/v1.8.2/apollo-configservice-1.8.2-github.zip
 * https://github.com/ctripcorp/apollo/releases/download/v1.8.2/apollo-adminservice-1.8.2-github.zip
 * https://github.com/ctripcorp/apollo/releases/download/v1.8.2/apollo-portal-1.8.2-github.zip
 * sha1 checksums
   - 3a11822a91bd1c7657896c74c3c997a37d010af5  apollo-configservice-1.8.2-github.zip
   - 31f19200301e1befef05a4ebdbeee9bfc408cfbf  apollo-adminservice-1.8.2-github.zip
   - 02b6853cb4968b143970161d5e675175cc8cf6f1  apollo-portal-1.8.2-github.zip

Maven 2 staging repository:

 * No maven artifact is published for this hotfix release

Release Tag :

 * https://github.com/ctripcorp/apollo/releases/tag/v1.8.2

Release CommitID :

 * https://github.com/ctripcorp/apollo/commit/7c486fa11e146d4bad65c69611614211bf56c42a

Guide to build the release from source :

 * https://www.apolloconfig.com/#/zh/deployment/distributed-deployment-guide?id=_2212-%e9%80%9a%e8%bf%87%e6%ba%90%e7%a0%81%e6%9e%84%e5%bb%ba

Voting will start now (2021/05/29) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by nisiyong** [comment]
+1

**Comment by zouyx** [comment]
+1 suggest put the release info into discusions block.

**Comment by nobodyiam** [comment]
> +1 suggest put the release info into discusions block.

Thanks for the suggestion and I think next time we could use the discussion block for this kind of votes.

**Comment by Anilople** [comment]
+1

**Comment by kezhenxu94** [comment]
> - https://github.com/ctripcorp/apollo/releases/download/untagged-06185836f1304b8d3eef/apollo-configservice-1.8.2-github.zip
> - https://github.com/ctripcorp/apollo/releases/download/untagged-06185836f1304b8d3eef/apollo-adminservice-1.8.2-github.zip
> - https://github.com/ctripcorp/apollo/releases/download/untagged-06185836f1304b8d3eef/apollo-portal-1.8.2-github.zip

These links seem to be expired

**Comment by nobodyiam** [comment]
> > * https://github.com/ctripcorp/apollo/releases/download/untagged-06185836f1304b8d3eef/apollo-configservice-1.8.2-github.zip
> > * https://github.com/ctripcorp/apollo/releases/download/untagged-06185836f1304b8d3eef/apollo-adminservice-1.8.2-github.zip
> > * https://github.com/ctripcorp/apollo/releases/download/untagged-06185836f1304b8d3eef/apollo-portal-1.8.2-github.zip
> 
> These links seem to be expired

fixed

**Comment by maoxiaoxiong** [comment]
404 ?

**Comment by nobodyiam** [comment]
@maoxiaoxiong I updated with the pre-release link, please check again.

**Comment by vdiskg** [comment]
+1

**Comment by lepdou** [comment]
+1

**Comment by JaredTan95** [comment]
1. sha1 checksums checked.

+1 binding for release 1.8.2.

**Comment by kezhenxu94** [comment]
+1 binding to release 1.8.2

I've checked 

- [x] shasum are correct for each zip file
- [x] deployed the services and test the main paths without problems.

I notice there is error in browser console, it seems to be the `nav.html` template is not rendered correctly in some places

<img width="574" alt="Screen Shot 2021-06-02 at 10 25 47" src="https://user-images.githubusercontent.com/15965696/120414651-e7662180-c38c-11eb-9919-7016dc39b563.png">

**Comment by nobodyiam** [comment]
@kezhenxu94 this was fixed in #3725. I think you could disable the cache and reload again.

**Comment by kezhenxu94** [comment]
> @kezhenxu94 this was fixed in #3725. I think you could disable the cache and reload again.

OK, then this release candidate looks good to me

**Comment by nobodyiam** [comment]
The vote to release Apollo version 1.8.2 has passed with two +1 binding, 5 +1 and zero -1.
